test/suite_test.go

```diff
@@ -35,50 +52,83 @@ func (e *e2e) SetupSuite() {
 	cwd, err := os.Getwd()
 	e.Nil(err)
 
-	buildDir := filepath.Join(filepath.Dir(cwd), "build")
+	parent_cwd := filepath.Dir(cwd)
+	os.Chdir(parent_cwd)
+	buildDir := filepath.Join(parent_cwd, "build")
 	e.Nil(os.MkdirAll(buildDir, 0o755))
 
 	e.kindPath = filepath.Join(buildDir, "kind")
 	if !util.Exists(e.kindPath) {
-		e.Nil(command.New(
+		e.run(
 			"curl", "-o", e.kindPath, "-fL",
 			"https://github.com/kubernetes-sigs/kind/releases/download/"+
 				kindVersion+"/kind-linux-amd64",
-		).RunSuccess())
+		)
 		e.Nil(os.Chmod(e.kindPath, 0o755))
 	}
 }
 
 // SetupTest starts a fresh kind cluster for each test
 func (e *e2e) SetupTest() {
+	// Deploy the registry
```

Comment:
Could we skip doing this and just load the image into the kind cluster? https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster

Reply:
Hah, yeah I change the implementation as suggested and will choose a pull policy of `Never`.

Reply:
This is one of my favorite features of `kind` :)